src/controller/CHIPDeviceController.cpp

```diff
@@ -1073,6 +1096,39 @@ void DeviceCommissioner::OnSessionEstablishmentTimeoutCallback(System::Layer * a
     reinterpret_cast<DeviceCommissioner *>(aAppState)->OnSessionEstablishmentTimeout();
 }
 
+CHIP_ERROR DeviceControllerInteractionModelDelegate::CommandResponseStatus(
+    const app::CommandSender * apCommandSender, const Protocols::SecureChannel::GeneralStatusCode aGeneralCode,
+    const uint32_t aProtocolId, const uint16_t aProtocolCode, chip::EndpointId aEndpointId, const chip::ClusterId aClusterId,
+    chip::CommandId aCommandId, uint8_t aCommandIndex)
+{
+    // Here we verify that the error code is not 0, since we may want to send more command to the device if we received success
+    // response.
+    VerifyOrReturnError(aProtocolCode != 0, CHIP_NO_ERROR);
+
+    // Generally IM has more detailed errors than ember library, here we always use 1 temporary, the actual handling of the
+    // commands should implement full IMDelegate.
+    IMDefaultResponseCallback(apCommandSender, EMBER_ZCL_STATUS_FAILURE /* 1 for error */);
```

Comment:
The comment is a bit unclear to me. Basically it says that IM has more error code, and because of this it does not dispatch the ZCL error code but use `1` for all errors ? This is a bit strange.